apiary.apib

```diff
@@ -3067,6 +3084,37 @@ Returns navigation information
         }
 
 
+## Course Certificates Status [/courses/{course_id}/certificates/status]
+
+It returns status for course certificates
+
+### Course Certificates Status [GET]
+
+Returns a JSON representation of certificate details for a specified course, also return status if we can request certificate generation for the course, possible statuses can be 'available', 'notavailable', 'generated', 'generating'
+
++ Response 200 (application/json)
+
+        {
+            "status": "available",
+            "generation_uri": "http://lms.mcka.local/api/server/courses/course-v1:edX+DemoX+Demo_Course/certificates/generate",
+        }
+
+
+## Generate Course Certificates [/courses/{course_id}/certificates/generate]
+
+It starts generation of certificates fot the specified course
```

Comment:
What's our use case here: Do we intend to call this once per course at some specific "end of course" date, or call this many times on an ongoing basis?